build-logic/src/main/kotlin/terasology-metrics.gradle.kts

```diff
@@ -25,13 +25,13 @@ dependencies {
     pmd("net.sourceforge.pmd:pmd-core:7.0.0-rc4")
     pmd("net.sourceforge.pmd:pmd-java:7.0.0-rc4")
 
-    testRuntimeOnly("ch.qos.logback:logback-classic:1.2.11") {
+    testRuntimeOnly("ch.qos.logback:logback-classic:1.2.12") {
         because("runtime: to configure logging during tests with logback.xml")
     }
     testRuntimeOnly("org.codehaus.janino:janino:3.1.7") {
         because("allows use of EvaluatorFilter in logback.xml")
     }
-    testRuntimeOnly("org.slf4j:jul-to-slf4j:1.7.36") {
+    testRuntimeOnly("org.slf4j:jul-to-slf4j:2.0.9") {
```

Comment:
Does that major version upgrade just work?

Reply:
looked like it, yes